packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_interface/webview_platform_controller.dart

```diff
@@ -103,20 +103,38 @@ abstract class WebViewPlatformController {
         "WebView clearCache is not implemented on the current platform");
   }
 
-  /// Evaluates a JavaScript expression in the context of the current page.
+  /// Evaluates a Javascript expression in the context of the current page.
```

Comment:
Comments shouldn't be changed to be wrong. We should leave the code consistently wrong until we mass-fix it in a breaking change, because it's difficult to change, but there's no reason for comments to be incorrect.

Please fix all the comments in the PR.